Makefile

```diff
@@ -15,9 +15,13 @@ VM_TYPE=hvm
 # For route53.tf
 PRIVATE_DOMAIN=$(CLUSTER_NAME).local
 
+# Development cluster variables, required by worker_dev module
+DEV_APP_FROM_PORT:= 8080
+DEV_APP_TO_PORT:= 8081
+
 # For gen-vpc-subnet-modules-tf.sh
 # Add all modules for which <module-name>-subnet.tf needs to be created
-VPC_SUBNET_MODULES=etcd,admiral,docker_registry,worker,elb,rds,aurora_db
+VPC_SUBNET_MODULES=etcd,admiral,docker_registry,worker,elb,rds,worker_dev,aurora_db
```

Comment:
@hazim1093 i think you need to look into this list; what is difference between "worker" & "worker_dev"? also I think we should update "etcd" to "etcd_dev"